src/app/wizard/step/provider-settings/provider/extended/openstack/default/template.html

```diff
@@ -65,4 +65,17 @@
       {{subnet.name}} ({{subnet.id}})
     </div>
   </km-combobox>
+
+  <mat-form-field fxFlex>
+    <mat-label>Allowed IP range for NodePorts</mat-label>
```

Comment:
Use title case.

Reply:
It is not resolved.

Reply:
I fixed this locally and marked it as resolved, since I was about to push my changes. But then found out a few other issues :)